File: qtile/.config/qtile/themes/colorschemes/catppuccin_macchiato/colors.py
from themes.core_colorschemes import catppuccin

c = catppuccin

other_colors = {
    "light": c["text"],
    "gray": c["base"],
    "dark": c["crust"],
    "too_dark": c["crust"],
    "font": c["text"],
    "bg": c["crust"],
    "border": c["crust"],
}

# Widget c
app_launcher_colors = {
    "fg": c["crust"],
    "bg": c["yellow"],
}

tray_colors = {
    "fg": c["crust"],
    "fg_icon": c["crust"],
    "bg": c["blue"],
}

windowname_colors = {
    "fg": c["text"],
    "bg": c["crust"],
}

windowcount_colors = {
    "fg": c["crust"],
    "bg": c["maroon"],
}

groupbox_colors = {
    "bg": c["red"],
    "bg_focus": c["base"],
    "bg_urgent": c["mauve"],
    "fg_active": c["base"],
    "fg_inactive": c["flamingo"],
    "fg_focus": c["red"],
    "fg_urgent": c["base"],
}

layout_colors = {
    "bg": c["maroon"],
}

volume_colors = {
    "fg": c["crust"],
    "bg": c["mauve"],
}

brightness_colors = {
    "fg": c["crust"],
    "bg": c["red"],
}

network_colors = {
    "fg": c["crust"],
    "bg": c["green"],
}

widgetbox_colors = {
    "fg": c["yellow"],
    "bg": c["yellow"],
}

battery_colors = {
    "fg": c["crust"],
    "fg_icon": c["rosewater"],
    "fg_low": c["crust"],
    "bg": c["rosewater"],
    "bg_icon": c["surface1"],
    "bg_low": c["red"],
}

time_colors = {
    "fg": c["crust"],
    "fg_icon": c["yellow"],
    "bg": c["yellow"],
    "bg_icon": c["surface1"],
}

date_colors = {
    "fg": c["crust"],
    "fg_icon": c["peach"],
    "bg": c["peach"],
    "bg_icon": c["surface1"],
}

client_colors = {
    "border": c["crust"],
    "border_focus": c["maroon"],
    "border_floating": c["rosewater"],
}

bar_colors = {
    "bg": c["crust"],
    "border": c["crust"],
}

File: awesome/.config/awesome/visual/themes/basic/colorschemes/nord/colors.lua
require("visual.colorschemes")

local colors = NORD

other_colors = {
  light = colors.white,
  gray = colors.black2,
  dark = colors.black,
  too_dark = colors.black0,

  font = colors.white,
  bg = colors.black0,
  border = colors.black,
}

---{{{ Widget Colors
app_launcher_colors = {
  fg = colors.black0,
  fg_icon = colors.black0,
  bg = colors.red1,
  border = colors.red1,
}

battery_colors = {
  fg = colors.black0,
  fg_icon = colors.green1,
  bg = colors.green1,
  bg_icon = colors.black1,
  border = colors.black1,
}

time_colors = {
  fg = colors.black0,
  fg_icon = colors.blue1,
  bg = colors.blue1,
  bg_icon = colors.black1,
  border = colors.black1,
}

date_colors = {
  fg = colors.black,
  fg_icon = colors.yellow1,
  bg = colors.yellow1,
  bg_icon = colors.black1,
  border = colors.black1,
}

tray_colors = {
  fg = colors.black,
  fg_icon = colors.black,
  bg = colors.red1,
  border = colors.red1,
}
---}}}

taglist_colors = {
  fg = colors.white,
  bg = colors.blue3,
  bg_focus = colors.black0,
  bg_urgent = colors.black0,
  fg_focus = colors.blue3,
  fg_occupied = colors.black0,
  fg_urgent = colors.red1,
  border = colors.blue4,
  hover = colors.white,
}

tasklist_colors = {
  fg = colors.blue4,
  fg_focus = colors.white,
  fg_minimize = colors.red1,
  bg = colors.black0,
  bg_focus = colors.blue4,
  bg_minimize = colors.red1,
  border = colors.black0,
}

titlebar_colors = {
  border = colors.black,
  border_focus = colors.green1,
}

bar_colors = {
  bg = colors.black0,
  border = colors.black0,
}
